app/models/user_block.rb

```diff
@@ -0,0 +1,16 @@
+class UserBlock < ApplicationRecord
+  belongs_to :blocker, foreign_key: "blocker_id", class_name: "User", inverse_of: :user_blocks
+  belongs_to :blocked, foreign_key: "blocked_id", class_name: "User", inverse_of: :user_blocks
```

Comment:
is it correct they both are the inverse of the same list?
also I've noticed there's no actual foreign key in the table definition...